packages/terra-form-select/CHANGELOG.md

```diff
@@ -4,8 +4,10 @@ ChangeLog
 Unreleased
 ----------
 ### Changed
-* updated package.json test scripts
+* WIP: splitting up terra-form-select
```

Comment:
Lets get this updated to be accurate about the changes. Let's also group all the change bullet points under the `Unreleased` section under 1 `### Changed` heading.